Makefile

```diff
@@ -105,6 +108,8 @@ migration-status:
 
 ci: covers phpunit cs npm-ci validate-app-config validate-campaign-config stan
 
+ci-with-coverage: covers phpunit-with-coverage cs npm-ci validate-app-config validate-campaign-config stan
```

Comment:
I think you need to add this to the [.PHONY targets](https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html)